src/app/component/functions/createLoanRequest.tsx

```diff
@@ -0,0 +1,136 @@
+// import {
```

Comment:
Deleted if not use, uncomment if i s used

Reply:
done